.env

```diff
@@ -33,7 +33,7 @@ APP_TYPE=component
 # The urls/ip's and hosts on wich this api is available
 
 TRUSTED_PROXIES=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-TRUSTED_HOSTS=^.+$
+TRUSTED_HOSTS=
```

Comment:
What is the difference between these 2 ^ ?

Reply:
I'm not sure, they are not used as far as I know, but Docker cannot build the first one